lms/static/styles/components/_StudentSelector.scss

```diff
@@ -1,29 +1,54 @@
 .StudentSelector {
   display: flex;
-  button {
-    background-color: transparent;
-    min-width: 80px;
-    border: none;
-    background-color: $grey-6;
-    transition: background-color 0.2s, opacity 0.2s;
-    outline: none;
-    &:hover {
-      cursor: pointer;
-      background-color: $grey-5;
-    }
-    &:disabled {
-      &:hover {
-        background-color: $grey-6;
-      }
-      cursor: default;
-      opacity: 0.5;
-    }
+}
+
+.StudentSelector-change-student {
+  svg {
```

Comment:
`SVGIcon` takes a `className` prop that you can use to attach a style-able property to the SVG; can you do that and then update this selector to be a class?

Reply:
Sure, this is also probably gonna be an area were the base component default style is going to give us this stuff out of the box.